### PR TITLE
Feat: ReplicationEnabledField for Storage Pool 

### PR DESCRIPTION
**Pull request by lalitpaliwal-ibm2**
With this PR, user will be able to see if storage pool is replication enabled or not while accessing storage pool/pools capacity APIs.
Jira: https://jsw.ibm.com/browse/POWERIAAS-10036

### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates OR Closes #0000

Output from acceptance testing:

```
$ make testacc TESTARGS='-run=TestAccXXX'

...
```

### CODE REVIEW

**Review by powervs-ibm — COMMENTED** [review]
Looks good. Could you run the acceptance test as well?

File: go.mod [hunk 1]
```diff
@@ -5,7 +5,7 @@ go 1.18
 require (
 	github.com/IBM-Cloud/bluemix-go v0.0.0-20230601050310-eecebfbff63e
 	github.com/IBM-Cloud/container-services-go-sdk v0.0.0-20230118060037-101bda076037
-	github.com/IBM-Cloud/power-go-client v1.2.4
+	github.com/IBM-Cloud/power-go-client v1.3.1
```
Comment (michaelkad) [inline go.mod:8]:
```suggestion
	github.com/IBM-Cloud/power-go-client v1.4.0
```
Was updated.

**Comment by lalitpaliwal-ibm2** [comment]
> Looks good. Could you run the acceptance test as well?

I have run the relevant acceptance test

File: go.mod (hunk 1 above)
Comment (lalitpaliwal-ibm2) [reply go.mod]: Sure,  updated power-go-client version

**Review by michaelkad — APPROVED** [review]

**Pull request merged**